README.md

```diff
@@ -8,24 +8,24 @@ The path follows the pattern: ``clients/{language}/github.com/{owner}/{repositor
 The ``{language}`` component of the path is the path-safe representation
 of the full language name which is mapped in [languages.json](./languages.json).
 
-Each client's JSON object represents the details displayed on the [clients documentation page](https://placeholderkv.io/docs/clients).
+Each client's JSON object represents the details displayed on the [clients documentation page](https://valkey.io/docs/clients).
 
-For example [clients/python/github.com/placeholderkv](./clients/python/github.com/placeholderkv/placeholderkv-py.json):
+For example [clients/python/github.com/valkey](./clients/python/github.com/valkey/valkey-py.json):
```

Comment:
There is no client with that name. Can you find another example which actually exists? Look at redis.io website and the files under clients/ for inspiration.

Reply:
Both of those are great ideas, but outside the scope of this PR.